pkg/controller/util/cluster_util.go

```diff
@@ -45,6 +45,11 @@ const (
 	KubeAPIBurst            = 30
 	KubeconfigSecretDataKey = "kubeconfig"
 	getSecretTimeout        = 1 * time.Minute
+
+	LeaseDuration = 15 * time.Second
```

Comment:
I think more descriptive names are suggested here, e.g. `DefaultLeaderElectionLeaseDuration`.  Reusing the names from `LeaderEelectionConfiguration` doesn't make sense to me because the context of being included in the struct is lost.

Reply:
Updated.